### DISCUSSION TITLE
How can i run this?

### DISCUSSION THREAD

**Discussion by bengitrombone** [NebulaServices/Nebula#158] [General]
How can i run this

**Comment by Scaratech** [comment]
wdym

**Comment by 0mhx** [comment]
What do you want to run this on?
There are clear instructions for this.

**Comment by SchoolTARMaster** [comment]
> How can i run this

You smarty, the instructions are clear on the repo.

**Comment by SchoolTARMaster** [comment]
just make sure you install npm, pnpm, and git
